scripts/compile_bundle.sh

```diff
@@ -153,31 +156,20 @@ unset LLVM_DIR
 unset USE_LLVM
 python -m pip uninstall -y mkl-static mkl-include
 python -m pip install --force-reinstall dist/*.whl
-git config --global --unset safe.directory
 #  TorchVision
 cd ../vision
-git config --global --add safe.directory `pwd`
-conda install -y libpng jpeg
 python setup.py clean
 python setup.py bdist_wheel 2>&1 | tee build.log
 python -m pip install --force-reinstall --no-deps dist/*.whl
-python -m pip install Pillow
-git config --global --unset safe.directory
 #  TorchAudio
 cd ../audio
-git config --global --add safe.directory `pwd`
-conda install -y bzip2
```

Comment:
why these conda install commands are removed?

Reply:
not everybody is using conda and this script should focus on compiling the whl files. it fails early if the requirements are not met and the user can take the needed actions to fulfill them. it's simply more portable.